scripts/get-buildconfig.ps1

```diff
@@ -90,6 +90,33 @@ if ("" -eq $Tls) {
         $Tls = "schannel"
     } else {
         $Tls = "quictls"
+        try {
+            # If no Tls was specified, try to guess it based on default OpenSSL version
+            # This is more complicated in attempt to silently deal with missing quictls executable
+            #
+            $pinfo = New-Object System.Diagnostics.ProcessStartInfo
+            $pinfo.FileName = "openssl"
+            $pinfo.Arguments = "version"
+            $pinfo.RedirectStandardError = $true
+            $pinfo.RedirectStandardOutput = $true
+            $pinfo.UseShellExecute = $false
+
+            $p = New-Object System.Diagnostics.Process
+            $p.StartInfo = $pinfo
+            $p.Start() | Out-Null
+            $p.WaitForExit()
+
+            $version = $p.StandardOutput.ReadToEnd()
+            if ($version -like "OpenSSL 3.1*")
+            {
+                $Tls = "quictls"
+            }
+            else
+            {
+                $Tls = "openssl"
```

Comment:
I thought openssl is only available for v3.5? If we're going to default to openssl, I want to make sure everything is there.

Reply:
I'll fix it up to search for OpenSSL 3.5 or greater, but its kind of a lousy test, since a system library of openssl-3.x where x < 5, is still likely to be openssl (i.e. the quictls api extensions won't be present)